frontend/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -73,14 +51,14 @@ const interactiveMessage = (proposal, baseOptions = {}) => {
     [PROPOSAL_INVITE_MEMBER]: {
       [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to add new members to the group.', options),
       [PROPOSAL_VARIANTS.EXPIRING]: L('Proposal from {from} to add new members is expiring.', options),
-      [PROPOSAL_VARIANTS.ACCEPTED]: L('Added members to this group: {to}', options),
+      [PROPOSAL_VARIANTS.ACCEPTED]: L('Added members to this group: {group}', options),
```

Comment:
I'm confused by this message.

Shouldn't this say who was added instead of the group name?

Reply:
```
[PROPOSAL_INVITE_MEMBER]: {
      [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to add {member} to the group.', options),
      [PROPOSAL_VARIANTS.EXPIRING]: L('Proposal from {from} to add {member} is expiring.', options),
      [PROPOSAL_VARIANTS.ACCEPTED]: L('Proposal from {from} to add {member} is accepted.', options),
      [PROPOSAL_VARIANTS.REJECTED]: L('Proposal from {from} to add {member} is rejected.', options),
      [PROPOSAL_VARIANTS.EXPIRED]: L('Proposal from {from} to add {member} is expired.', options)
},
```

These are more precise, aren't they?

Reply:
> These are more precise, aren't they?

Yes, these seem much better 👍